src/directives/bindInput.ts

```diff
@@ -144,26 +144,35 @@ class BindInputDirective extends AsyncDirective {
 
     this.__lastValue = value;
 
-    if (isInputElement(element)) {
-      if (element.type === 'checkbox') {
-        element.checked = value === true;
-      } else {
-        element.value = String(value ?? '');
-      }
-    } else if (isSelectElement(element)) {
-      if (element.multiple) {
-        const valuesArray = Array.isArray(value) ? value : [value];
+    if (isSelectElement(element)) {
+      switch (element.type.toLowerCase()) {
+        case 'select-multiple':
```

Comment:
where is this from? a select should have a `multiple` attribute if it wants multiple values